assignment01/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head></head>

```

Comment:
this is where I would expect to see the title of the document and any links out to external js files (i.e. jQuery and your custom js).

Reply:
Okay, got it! I did not include my scripts within the <head></head> tags on my redux (that was more an attempt at a correct turn in), but I looked it it up and now I understand that is where they should go.